rep-2000.rst

```diff
@@ -306,6 +307,111 @@ Build System Support:
 - cmake
 - setuptools
 
+
+Dashing Diademata (May 2019 - May 2021)
+---------------------------------------------------------------
+
+Targeted platforms:
+
++--------------+-----------------------+------------------------------+------------------------+------------------------+
+| Architecture | Ubuntu Bionic (18.04) |     MacOS Sierra (10.12)     |   Windows 10 (VS2019)  | Debian Stretch (9) [s] |
++==============+=======================+==============================+========================+========================+
+|    amd64     |        Tier 1         |           Tier 2*            |          Tier 1*       |        Tier 3          |
++--------------+-----------------------+------------------------------+------------------------+------------------------+
+|    arm64     |        Tier 1         |                              |                        |        Tier 3          |
++--------------+-----------------------+------------------------------+------------------------+------------------------+
+|    arm32     |        Tier 2         |                              |                        |        Tier 3          |
```

Comment:
The arm32/Bionic cell needs a star since we won't provide package-level binaries for that platform: see #194.